internal/pkg/skuba/deployments/ssh/kured.go

```diff
@@ -47,3 +50,13 @@ func kuredDeploy(t *Target, data interface{}) error {
 	_, _, err = t.ssh("kubectl --kubeconfig=/etc/kubernetes/admin.conf apply -f /tmp/kured.d")
 	return err
 }
+
+func kuredLock(t *Target, data interface{}) error {
+	_, _, err := t.ssh("kubectl", "--kubeconfig=/etc/kubernetes/admin.conf", "-n", metav1.NamespaceSystem, "annotate", "ds", "kured", "weave.works/kured-node-lock='{\"nodeID\":\"manual\"}'")
```

Comment:
isn't this going to fail if the target is a worker since there is no `/etc/kubernetes/admin.conf` ?

Reply:
good point, this should only be run on a master

Reply:
i've used kubernetes client libraries to change the annotations now. so we're independent of the kubeconfig